src/Request.php

```diff
@@ -476,15 +476,17 @@ private function after_execute_actions( $response, $key = null ) {
 		$query     = null;
 		$operation = null;
 		$variables = null;
+		$query_id  = null;
 
 		if ( $this->params instanceof OperationParams ) {
 			$operation = $this->params->operation;
 			$query     = $this->params->query;
+			$query_id  = $this->params->queryId;
 			$variables = $this->params->variables;
 		} elseif ( is_array( $this->params ) ) {
-
 			$operation = $this->params[ $key ]->operation ?? '';
 			$query     = $this->params[ $key ]->query ?? '';
+			$query_id  = $this->params[ $key ]->queryId ?? '';
```

Comment:
I know it's the pattern for the other vars, but maybe we should set this to `?? null` instead of an empty string?

Reply:
👌